src/components/slider/slider.tsx

```diff
@@ -124,16 +124,48 @@ export class Slider {
             return;
         }
 
+        const inputElement: HTMLInputElement = element.querySelector('input');
+        const value = this.getValue();
+
         /*
         For some reason the input element's `value` attribute is removed
         (probably by Stencil) when the element is first rendered. But if the
         attribute is missing when MDCSlider is initialized (MDC v11.0.0),
         MDCSlider crashes.
         So we add the attribute right before initializing MDCSlider. /Ads
         */
-        element
-            .querySelector('input')
-            .setAttribute('value', `${this.multiplyByFactor(this.getValue())}`);
+        inputElement.setAttribute('value', `${this.multiplyByFactor(value)}`);
+
+        /*
+        When creating the `mdcSlider` component, its important that the value set in
+        the input field obeys the range and the step size.
+
+        The MDCSlider will throw an exception unless the value in the input element
+        is dividible by the step value and is in the provided range.
+        If an exception occurs, this component will crash and it will be impossible to change
+        its value.
+        The logic below removes ensures that the component will render even though the
+        provided value is wrong.
+        This could be considered wrong, but it at least fixes such that it's possible
+        to change the value from the UI.
+        */
+        const greaterThanMin = value > this.valuemin;
+        const lessThanMax = value < this.valuemax;
+        const dividableByStep = value % this.step === 0;
+        const delta = 0.01;
```

Comment:
If the range is between 0-100 (`this.valuemin` - `this.valuemax` ), is this range 0.01?

Reply:
delta is something made up by me just to ensure that we can initialize the slider with valid values. It could have been any value really.

Looking back at the problem, I think it's only needed when value is greaterThanMax.

Reply:
delta was needed in both `greaterThanMax` and `lessThanMax`. It crushes when you have a data for example between  1-100

Reply:
Thanks for looking into this! I've created a fixup with your suggestions